cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,13 @@
+describe('login', () => {
+  it('open up the login form and successfully login with user credentials and show profile page', () => {
+    cy.visit('https://karlpnord.github.io/social-media-client/');
```

Comment:
```suggestion
    cy.visit('/');
```

You should be testing the "local" code that is in a branch, rather than the "remote" deployed code from main. Cypress runs its own live server :)